centreon/www/include/Administration/myAccount/formMyAccount.php

```diff
@@ -418,7 +422,7 @@ function myReplace()
 $sessionKeyFreeze = 'administration-form-my-account-freeze';
 
 if ($form->validate()) {
-    updateContactInDB($centreon->user->get_id());
+    updateContactInDBIfLocal($cct['contact_id']);
```

Comment:
The method name suggests that the update is only performed if the user is 'local', which is not true.

Reply:
i suggest those name for the method : progressiveContactUpdate - contextualContactUpdate -incrementalContactUpdate 
Which one do you think is the best ?